bittensor/config.py

```diff
@@ -215,7 +219,7 @@ def __init__(
                             cmd_parser._defaults.clear()  # Needed for quirk of argparse
 
         ## Reparse the args, but this time with the defaults as argparse.SUPPRESS
-        params_no_defaults = config.__parse_args__(
+        params_no_defaults = self.__parse_args__(
```

Comment:
Apparently there's a params_no_defaults we can use as a baseline instead to prevent this.

merged_config = {
**self.defaults_config,
...
**self.params_no_defaults_config,
}

Reply:
it's already like that.